### ISSUE TITLE
[Youtube] Make subtitles selection more accessible

### ISSUE THREAD

**Issue by R0M4N0F5K7** [futo-org/grayjay-android#1447]
### Use Cases

As a user I want to access available subtitles regularly and quickly. Depending on how loud my surroundings are, I am eating crunchy snacks, I can use headphones or the speakers or of the video is in a foreign language (YouTube almost always provides auto generated English subtitles these days), I want to quickly check for and turn subtitles on of off. Currently they are below the long list of video and audio options, which are selected based on my default preferences anyway. 

### Proposal

The easiest fix world be to just move the subtitle selection up, above the quality settings and below the playback speed settings as those settings are more likely to be changed in a video-to-video basis. The quality preferences are configured in the settings anyway. If there are really people constantly manually switching the video quality it might be worth considering splitting "quality" settings (video and audio source) and "payback" settings (playback speed, subtitle languages) or allowing the user to set the order themselves.

**Comment by alistair-broomhead** [comment]
As a disabled user who wants to move over from NewPipe this would be huge. It would be good to have a setting to have subtitles on by default, but I'm happy to open a new issue for that if it's beyond the scope of this enhancement.

**Comment by JM-PT** [comment]
Agree, its very annoying to need 2 clicks to add subtitles or change speed.

Should also have a default setting

**Comment by alistair-broomhead** [comment]
I just want to add that when I watch a video from Nebula it shows subtitles by default. I don't know whether this is something it gets from my Nebula preferences or just the default Grayjay configuration for that source, but it makes watching videos from there through your app much more pleasant than YouTube videos where I have to hunt for the setting every time it switches video

**Comment by alistair-broomhead** [comment]
Currently it feels like https://github.com/futo-org/grayjay-plugin-youtube/issues/6 makes this irrelevant, as I'm no longer managing to show subtitles at all on any YouTube videos

**Comment by Hafidzgtg** [comment]
Hello🙏🏻....
I have the same issue. I want an auto-translate feature to Indonesian language in this application so it is easier to understand, and it should follow the generated subtitles to Indonesian in the video later. Please fix this. Thanks!🙏🏻
